_source/_posts/2021-03-01-oauth-refcard-patterns-antipatterns.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: blog_post
+title: "OAuth Patterns and Anti-Patterns - a DZone Refcard"
+author: aaron-parecki
+by: advocate
+communities: [security]
+description: "I'm happy to announce a brand new OAuth refcard: OAuth Patterns and Anti-Patterns"
+tags: [oauth, security, refcardz, dzone, openid, oidc]
+tweets:
+- "Get our new cheat sheet: OAuth Patterns and Anti-Patterns"
+- "\"OAuth Patterns and Anti-Patterns\" - an OAuth refcard by @aaronpk"
```

Comment:
Do we know if whatever-processes-this-line works with this style of quote escaping?

Reply:
well I didn't see a YAML error so I assume so. I don't see the resulting tweets in `feed-tweets.csv` but I'm also not sure the criteria for what tweets appear in that feed.